Snippets/Sqs/Sqs_All/QueueCreation/CloudFormationHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace SqsAll.QueueCreation
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+
+    public class CloudFormationHelper
+    {
+        /// <summary>
+        /// CloudFormation doesn't allow comments found at the beginning and end of template for documentation in the template body.
+        /// </summary>
+        public static string ConvertToValidJson(string templatePath)
+        {
+            var allLines = File.ReadAllLines(templatePath);
+            return string.Join(Environment.NewLine, allLines.Skip(1).Take(allLines.Length - 2));
```

Comment:
Should we make a note that this code expects a docs comment at the beginning and at the end of the script? Because if anyone moves the comment `Skip(1)` and `Take(allLines.Length-2)` would no longer be necessary? Or better to strip the lines that start with `//`?

Reply:
I think comment is better. And it's already [there](https://github.com/Particular/docs.particular.net/pull/3658/files/e1ec7c92b89f217a2d9d77f06f48ba37d5c31d63#diff-97bcb11ad7ebe93165a4de31358833b6R10).